pkg/api/componentreadiness/component_report.go

```diff
@@ -350,7 +388,7 @@ func (c *componentReportGenerator) GenerateComponentReportTestStatus() (crtype.R
 }
 
 // getCommonTestStatusQuery returns the common query for the higher level summary component summary.
-func (c *componentReportGenerator) getCommonTestStatusQuery(allJobVariants crtype.JobVariants, isSample bool) (string, string, []bigquery.QueryParameter) {
+func (c *componentReportGenerator) getCommonTestStatusQuery(allJobVariants crtype.JobVariants, isSample, isFallback bool) (string, string, []bigquery.QueryParameter) {
```

Comment:
Our production redis pod looks to run pretty consistently around 500mb.

Locally with a little testing I'm around 2.2gb. I believe this is with all main views cached so that number isn't horrible, but it looks like you will need to bump the redis memory request up from 1gb to maybe 5-10, otherwise we'll be cache missing too often as things may get evicted.

Reply:
Let's discuss 135a2c0711cf31b74918a9267f6e83f1a13c3992 and possibly a separate redis deployment for these large caches so we don't impact the main report cache.  Having redis in the persistent cache layer is optional but does speed things up.  So nice to have but if something is evicted we can still get it quicker from the persistent backend than completely redoing the query.